Gemfile

```diff
@@ -1,3 +1,7 @@
 source 'https://rubygems.org'
 gem 'github-pages'
+<<<<<<< HEAD
```

Comment:
These merge conflict markers need to be resolved - otherwise, the site won't build